build/build/src/enso.rs

```diff
@@ -73,11 +73,21 @@ impl BuiltEnso {
     }
 
     pub async fn run_benchmarks(&self, opt: BenchmarkOptions) -> Result {
-        self.cmd()?
-            .with_args(["--jvm", "--run", self.paths.repo_root.test.benchmarks.as_str()])
+        let filename = format!("enso{}", if TARGET_OS == OS::Windows { ".exe" } else { "" });
+        let enso = self
+            .paths
+            .repo_root
+            .built_distribution
+            .enso_engine_triple
+            .engine_package
+            .bin
+            .join(filename);
+        let benchmarks = Command::new(&enso)
+            .args(["--jvm", "--run", self.paths.repo_root.test.benchmarks.as_str()])
             .set_env(ENSO_BENCHMARK_TEST_DRY_RUN, &Boolean::from(opt.dry_run))?
             .run_ok()
-            .await
+            .await;
+        Ok(benchmarks?)
```

Comment:
Getting Rust to compile isn't easy. When one finally succeeds, one gets a [linter warning](https://github.com/enso-org/enso/actions/runs/9803687435/job/27070203607#step:7:142). Supressing for now 83f31be7b1, CCing @kazcw

Reply:
Fixed by Kaz. Thank you: [13c9518](https://github.com/enso-org/enso/pull/10374/commits/13c95186f57492cee0ae232231c6d81017c5d5eb)